Crashing from death 1.16.3

Minecraft Version: 1.16.3

Forge Version: 34.1.42

Mod Version: Cyclic-1.16.3-0.9.1

Single Player or Server: singeplayer

Describe problem (what you were doing; what happened; what should have happened):
is not giving a crash report but i tested with and without cylic with cylic if you die not mater how you die you will CTD
here is info from log that lead me here before finding out others had this issue this is not a crash report as it does not create one just ctd exit code 0

also this may be a duplicate issue as but it also looked like that was fixed in hotfix not sure if this is the same crash.



Log file link:
https://pastebin.com/c2LGtjqG this is from console in singeplayer game

https://www.dropbox.com/s/m043yt9ej5d55rd/latest.log?dl=0

i think this is the main reason for crash Caused by: java.lang.NullPointerException
	at com.lothrazar.cyclic.enchant.EnchantXp.handleEntityDropEvent(EnchantXp.java:37)

--- a/src/main/java/com/lothrazar/cyclic/enchant/EnchantXp.java
+++ b/src/main/java/com/lothrazar/cyclic/enchant/EnchantXp.java
@@ -34,6 +34,9 @@ public class EnchantXp extends EnchantBase {
 
   @SubscribeEvent
   public void handleEntityDropEvent(LivingExperienceDropEvent event) {
+    if (event.getAttackingPlayer() == null) {
+      return;
+    }
     int level = getCurrentLevelTool(event.getAttackingPlayer().getHeldItemMainhand());
     if (level <= 0) {
       return;

